docs/developers/gas-and-fees/egld-transfers.md

```diff
@@ -0,0 +1,36 @@
+---
+id: egld-transfers
+title: EGLD transfers
```

Comment:
might rename to move balance transfers

Reply:
That's indeed the term we use, in the protocol. I will add it in the title, in parenthesis, but also keep the more "practical" name. Is it all right?

Reply:
it is ok